packages/frontend/src/build/config/common.ts

```diff
@@ -28,7 +28,7 @@ export const common: Omit<Config, 'backend'> = {
     stages: true,
     buildAllProjectPages: false,
     upcomingRollups: true,
-    multisigReport: true,
+    multisigReport: false,
```

Comment:
This does not look like disabling the topbar

Reply:
Check out the preview. If I don't understand something we'll talk :+]